app/forms/candidate_interface/degrees/form_builder.rb

```diff
@@ -0,0 +1,117 @@
+module CandidateInterface
```

Comment:
All this used to live directly on the DegreeWizard, so just moved it into its own class. It is tested implicitly in the `base_form_spec.rb` (which used to be the `degree_wizard_spec.rb`). let me know if you think it would be worth moving it to its own unit test at this point.